packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -0,0 +1,262 @@
+};
+
+const generateCrumbOverrides = () => {
+  const overrides = [
+    {
+      label: 'Definitions',
+      linkTo: '/monitor/cloudpulse/alerts/definitions',
+      position: 1,
+    },
+    {
+      label: 'Details',
+      linkTo: `/monitor/cloudpulse/alerts/definitions/create`,
+      position: 2,
+    },
+  ];
+  return { newPathname: '/Definitions/Details', overrides };
+};
+
+export const CreateAlertDefinition = React.memo(() => {
+  const {
+    data: engineOptions,
+    isError: engineOptionError,
+    isLoading: engineOptionLoading,
+  } = useDatabaseEnginesQuery(true);
+
+  // const {
+  //   data: notificationChannels,
+  //   isError: notificationChannelError,
+  //   isLoading: notificationChannelLoading,
+  // } = useNotificationChannels();
+
+  const history = useHistory();
+  const alertCreateExit = () => {
+    const pathParts = location.pathname.split('/');
+    pathParts.pop();
+    const previousPage = pathParts.join('/');
+    history.push(previousPage);
+  };
+
+  const formMethods = useForm<CreateAlertDefinitionPayload>({
+    defaultValues: initialValues,
+    mode: 'onBlur',
+    resolver: yupResolver(createAlertDefinitionSchema),
+  });
+
+  const [maxScrapeInterval, setMaxScrapeInterval] = React.useState<number>(0);
+  // const [openAddNotification, setOpenAddNotification] = React.useState(false);
+  // const [notifications, setNotifications] = React.useState<
+  //   NotificationChannel[]
+  // >([]);
+  const {
+    control,
+    formState,
+    handleSubmit,
+    setError,
+    setValue,
+    watch,
+  } = formMethods;
+  const { enqueueSnackbar } = useSnackbar();
+  const { mutateAsync: createAlert } = useCreateAlertDefinition(
+    watch('service_type')
+  );
+
+  // const onChangeNotifications = (notifications: NotificationChannel[]) => {
+  //   const notificationTemplateList = notifications.map(
+  //     (notification) => notification.id
+  //   ); 
+  //   setValue('channel_ids', notificationTemplateList);
+  // };
+
+  // const onSubmitAddNotification = (notification: NotificationChannel) => {
+  //   const newNotifications = [...notifications, notification];
+  //   const notificationTemplateList = newNotifications.map(
+  //     (notification) => notification.id
+  //   );
+  //   setValue('channel_ids', notificationTemplateList);
+  //   setNotifications(newNotifications);
+  //   setOpenAddNotification(false);
+  // };
+
+  const onSubmit = handleSubmit(async (values) => {
+    try {
+
+      await createAlert(values);
+      enqueueSnackbar('Alert successfully created', {
+        variant: 'success',
+      });
+      alertCreateExit();
+    } catch (errors) {
+      for (const error of errors) {
+        if (error.field) {
+          setError(error.field, { message: error.reason });
+        } else {
+          setError('root', { message: error.reason });
+        }
+      }
+    }
+  });
+
+  const { newPathname, overrides } = generateCrumbOverrides();
```

Comment:
here is possibility of using memoization, because generateCrumbOverrides function will give same result always.